tables.md

```diff
@@ -0,0 +1,198 @@
+
+| `serverlogs` ||
+| :---: | --------- |
+| | There two kind of Tableau Server log files. One set of processes log in JSON format and an other set of them just in plain text files. This table contains the raw log lines from JSON log files. |
+
+| `threadinfo` ||
+| :---: | --------- |
+| | Contains raw cpu and memory metric values received from the Tableau Server nodes. |
+
+
+
+| Non-historized repository tables | |
+| :---: | --- |
+|| These tables are polled from the Tableau server node that has the Repository on it. The content of these tables are streamed which means that only changed or newly created records are polled. |
+| **Tables:** |
+|| `async_jobs` |
+|| `background_jobs` |
+|| `hist_capabilities` |
+|| `hist_comments` |
+|| `hist_configs` |
+|| `hist_data_connections` |
+|| `hist_datasources` |
+|| `hist_groups` |
+|| `hist_licensing_roles` |
+|| `hist_projects` |
+|| `hist_schedules` |
+|| `hist_sites` |
+|| `hist_tags` |
+|| `hist_tasks` |
+|| `hist_users` |
+|| `hist_views` |
+|| `hist_workbooks` |
+|| `historical_disk_usage` |
+|| `historical_event_types` |
+|| `historical_events` |
+|| `http_requests` |
+
+
+| Historized repository tables | |
+| :---: | --- |
+|| These tables are polled from the Tableau repository tables with similar names (without the `h_` prefix. The difference is that Palette reporting takes notes on which values were active at a certain point of time (with an algorithm called SCD. )  |
+| **Tables:** |
+|| `h_capabilities` |
+|| `h_capability_roles` |
+|| `h_core_licenses` |
+|| `h_customized_views` |
+|| `h_data_connections` |
+|| `h_datasources` |
+|| `h_extracts` |
+|| `h_group_users` |
+|| `h_groups` |
+|| `h_monitoring_dataengine` |
+|| `h_monitoring_postgresql` |
+|| `h_next_gen_permissions` |
+|| `h_permission_reasons` |
+|| `h_projects` |
+|| `h_schedules` |
+|| `h_sites` |
+|| `h_subscriptions` |
+|| `h_subscriptions_customized_views` |
+|| `h_subscriptions_views` |
+|| `h_subscriptions_workbooks` |
+|| `h_system_users` |
+|| `h_tasks` |
+|| `h_user_default_customized_views` |
+|| `h_users` |
+|| `h_views` |
+|| `h_workbooks` |
+
+| Internally used "technical" tables | |
+| :---: | --- |
+|| These tables are needed for variouos technical reasons. |
+| **Tables:** |
+| This is needed to handle Tableau sessions running around UTC midnight. As our reporting uses UTC midnight as a separator, this is needed to handle sessions that overlap two days. | `cross_utc_midnight_sessions` |
+| External tables. These tables are used to import data from the csv files sent by the Palette Insight Agents to the database. | `ext_*` |
+| Partitions. Palette Insight utilizes the partitioning feature of Pivotal Greenplum Database and these tables are the actual tables of the partitions. | `_prt_*` |
+| Stage tables. These are loaded during the reporting and after their content is filled that is fully moved to the matching `p_` table.| `s_*` |
```

Comment:
I would rephrase this sentence like this:

"These are loaded during the reporting, and after their content is filled fully, then its records are moved to the matching `p_` table."

Or something like that.

Reply:
Yeah but fully would me completely different thing that way and that's not what I meant.